jax/_src/custom_derivatives.py

```diff
@@ -1150,12 +1150,12 @@ def custom_gradient(fun):
 
   >>> @jax.custom_gradient
   ... def f(x):
-  ...   return x ** 2, lambda g: (g * x,)
+  ...   return x ** 2, lambda g: (g * 2 * x,)
```

Comment:
Hi! This is on purpose to show that you can return any gradient you want. Can you add a comment instead?